src/test/java/com/spotify/folsom/client/OpCodeTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2015 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.spotify.folsom.client;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class OpCodeTest {
+
+  @Test
+  public void testGetKind() throws Exception {
```

Comment:
would it make sense to use an EnumSet, created using EnumSet.allOf() instead.
The advantage is less code + don't have to amend test if we add OpCodes.